src/Protocol/ActivityPub.php

```diff
@@ -359,7 +358,7 @@ public static function createPermissionBlockForItem($item)
 
 		$actor_profile = APContact::getProfileByURL($item['author-link']);
 
-		$terms = Term::tagArrayFromItemId($item['id']);
+		$terms = Term::tagArrayFromItemId($item['id'], TERM_MENTION);
```

Comment:
Why having removed `TERM_HASHTAG`? The `foreach` on line 371 doesn't make sense anymore.